demos/browser/app/meeting/meeting.ts

```diff
@@ -549,11 +554,16 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
     }
   }
 
+
   async initializeMeetingSession(configuration: MeetingSessionConfiguration): Promise<void> {
-    const logger = new ConsoleLogger('SDK', LogLevel.DEBUG);
-    const deviceController = new DefaultDeviceController(logger);
+    if (!(DemoMeetingApp.BASE_URL.includes('localhost') || DemoMeetingApp.BASE_URL.includes("127.0.0.1"))) {
```

Comment:
A builder may map other domain names to 127.0.0.1. In macOS, for example, you can add a custom domain `me` to `/etc/hosts`. This `if` condition will be false.

```
127.0.0.1 me
127.0.0.1 localhost
```

Instead, you want to introduce an environment variable. We use an environment variable `npm_config_app` to choose which demo app under `demos/browser/app` to build. You can search how `npm_config_app` is being passed in our repo.

https://github.com/aws/amazon-chime-sdk-js/blob/013dee3adb7f92fbc05919794cb9e8d542f108dc/demos/browser/webpack.config.js#L9